src/components/calculator.css

```diff
@@ -1,4 +1,4 @@
-*,
+/* *,
```

Comment:
- [ ] Kindly please remove all lines of old commented pieces of code, so the project gets cleaner 💪

Reply:
done